README.md

```diff
@@ -65,6 +65,8 @@ $ chcon -R system_u:object_r:admin_home_t:s0 docker-elk/
 
 ### Bringing up the stack
 
+**Note**: In case you switched branch / already have (any|all) of these containers / updated a base image - you may need to run `docker-compose build` first
```

Comment:
Containers are irrelevant, only images matter, so I'd remove _"/ already have (any|all) of these containers /"_ altogether.